app/representers/api/media_resource_representer.rb

```diff
@@ -2,10 +2,14 @@ class Api::MediaResourceRepresenter < Roar::Decorator
   include Roar::JSON
   include HalApi::Representer::FormatKeys
 
-  property :href
-  property :original_url, writeable: false
+  property :href, readable: false
+  property :file_name
   property :mime_type, as: :type
   property :file_size, as: :size
   property :duration
   property :status, writeable: false
+
+  # TODO: deprecate
+  property :href
```

Comment:
Duplicate of line 5?

Reply:
This is minor, but probably worth fixing.

Reply:
Ah, when I was initially going to remove href from the public API, I left it as writeable above.  So you can PUT media via the public endpoint, but cannot read the href.

Then I figured out we need some DTR changes (move it to use the auth endpoints) before we can do this.  So I added it back again.  I'll comment out line 5 to clarify.